Code/Engine/RendererCore/Meshes/Implementation/MeshComponentBase.cpp

```diff
@@ -192,13 +192,19 @@ void ezMeshComponentBase::OnMsgExtractRenderData(ezMsgExtractRenderData& msg) co
       pRenderData->FillBatchIdAndSortingKey();
     }
 
+    bool bDontCacheYet = false;
+
     // Determine render data category.
     ezRenderData::Category category = m_RenderDataCategory;
     if (category == ezInvalidRenderDataCategory)
     {
       if (hMaterial.IsValid())
       {
         ezResourceLock<ezMaterialResource> pMaterial(hMaterial, ezResourceAcquireMode::AllowLoadingFallback);
+
+        if (pMaterial.GetAcquireResult() == ezResourceAcquireResult::LoadingFallback)
```

Comment:
@C-Core kind of an important fix. I already fixed this in the greybox component previously, but this way is more elegant IMO, so I applied the same fix there now.